next/app/components/common/GenerateButton/GenerateButton.module.css

```
@@ -0,0 +1,42 @@
+.generateButton {
+  background-color: #edc3e7;
+  border: 1px solid black;
+  color: black;
+  padding: 10px 20px;
```

デザインに沿うと、borderは2px、水平のpaddingは30pxくらいが良さそうです。完全な黒は避けて、今回は`#2C3844`にしましょう。
```suggestion
  border: 2px solid #2C3844;
  color: #2C3844;
  padding: 10px 30px;
```